CMakeLists.txt

```diff
@@ -187,6 +187,10 @@ if(NOT CONFIG_RTTI)
   )
 endif()
 
+if(NOT CONFIG_NATIVE_APPLICATION)
```

Comment:
not native, meaning host, meaning posix.

Reply:
Exactly. My point was that this should live in host-gcc rather than being common for all in the root CMakeLists.txt

Reply:
Funny that: PR #13606

Reply:
I thought you meant: 

> (not native), meaning host, meaning posix

But in fact you meant:

> not (native, meaning host, meaning posix)
correct?

BTW there's an on-going slack discussion about the different regressions caused by this.